llvm/lib/Target/AArch64/AArch64ISelLowering.cpp

```diff
@@ -14688,6 +14695,40 @@ static SDValue tryToConvertShuffleOfTbl2ToTbl4(SDValue Op,
                       Tbl2->getOperand(1), Tbl2->getOperand(2), TBLMask});
 }
 
+SDValue
+AArch64TargetLowering::LowerEXTEND_VECTOR_INREG(SDValue Op,
```

Comment:
nit: It's a minor thing, but `LowerEXTEND_VECTOR_INREG` sounds like it handles all kinds of extend, including zero-extend. Given that ANY_EXTEND can include SIGN_EXTEND is it worth just renaming this to `LowerSIGN_EXTEND_VECTOR_INREG` to distinguish this from the zero-extend case?